run_splunk_container.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+source ./preview.env
+
+SPLUNK_PASSWORD="Password"
+SPLUNK_START_ARGS="'--accept-license'"
+URL="http://localhost:8089"
+
+build_and_run_image() {
+    # Ensure the IMAGE_NAME exists
+    [[ -z "$IMAGE_NAME" ]] && echo "Error: IMAGE_NAME ENV is not set!" && exit 1
+
+    echo "Building Podman image..."
+    podman build -t "$IMAGE_NAME" . && echo "Image '$IMAGE_NAME' built successfully!" || { echo "Error: Failed to build the image."; exit 1; }
+
+    echo "Running Podman container..."
+    podman run -d -e SPLUNK_PASSWORD="$SPLUNK_PASSWORD" -e SPLUNK_START_ARGS="$SPLUNK_START_ARGS" -p 8000:$WEB_PORT -p 8089:$API_PORT "$IMAGE_NAME" && echo "Container from image '$IMAGE_NAME' started successfully!" || { echo "Error: Failed to start the container."; exit 1; }
+}
+
+wait_for_splunk_cli() {
+    TIMEOUT=120
+    SLEEP_INTERVAL=5
+    echo "Waiting for Splunk CLI to come up"
+    start_time=$(date +%s)
+
+    while [[ "$(curl -s -o /dev/null -w %{http_code} -u admin:$SPLUNK_PASSWORD --insecure $URL)" != "200" && $(( $(date +%s) - start_time )) -lt $TIMEOUT ]]; do
+        sleep $SLEEP_INTERVAL
+    done
+
+    [[ "$(curl -s -o /dev/null -w %{http_code} -u admin:$SPLUNK_PASSWORD --insecure $URL)" == "200" ]] && echo "Splunk CLI is Up" || echo "Timeout reached while waiting for Splunk CLI to come up"
+}
+
+is_splunk_up() {
+    [[ $(curl -k -u admin:$SPLUNK_PASSWORD -s -o /dev/null -w "%{http_code}" $URL/services/server/info) == 200 ]]
+}
```

Comment:
Why do we need this if we already have a function that waits for splunk to be up? The only major difference between the curl commands is the path (`http://localhost:8089` vs `http://localhost:8089/services/server/info`). Why do we need to check both?

Reply:
I will remove it.